cogboard-webapp/src/components/widgets/types/PullRequestReminderWidget/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+
+import { formatPullRequestTitle, preparePullRequestArray } from './helpers.js';
+
+import { StyledNoItemsInfo } from '../../../Widget/styled';
+import { PullRequestContainer, PullRequestLink } from './styled';
+
+const PullRequestReminderWidget = ({ repositoryHub, pullRequests }) => {
+  pullRequests = preparePullRequestArray(repositoryHub, pullRequests);
+
+  if (!pullRequests) {
+    return (
+      <StyledNoItemsInfo>
+        <InfoOutlinedIcon fontSize="large" />
+        <p>API rate limit exceeded!</p>
+      </StyledNoItemsInfo>
+    );
+  }
+
+  return (
+    <>
+      {pullRequests.length > 0 ? (
+        pullRequests.map(({ id, title, url }, index) => (
+          <PullRequestLink key={id} href={url} target="_blank">
+            <PullRequestContainer>
+              {`${index + 1}. ${formatPullRequestTitle(title)}`}
+            </PullRequestContainer>
+          </PullRequestLink>
+        ))
+      ) : (
+        <StyledNoItemsInfo>
```

Comment:
I think that this part will never be rendered because of condition in line 14

Reply:
Github API returns empty array when there is no pull requests in a given reposiotry, so the widget will display "No Pull Requests", but if path to repository is invalid or API limit is reached it will return nothing, so widget will display "API rate limit exceeded!"